pkg/protocol/header.go

```diff
@@ -117,6 +117,7 @@ type ResponseHeader struct {
 	statusCode         int
 	contentLength      int
 	contentLengthBytes []byte
+	contentEncoding    []byte
```

Comment:
According to the naming style before and after, it is better as "contentEncodingBytes"?